x/oracle/internal/keeper/params.go

```diff
@@ -41,6 +41,24 @@ func (k Keeper) Whitelist(ctx sdk.Context) (res types.DenomList) {
 	return
 }
 
+// SlashFraction returns oracle voting penalty rate
+func (k Keeper) SlashFraction(ctx sdk.Context) (res sdk.Dec) {
+	k.paramSpace.Get(ctx, types.ParamStoreKeySlashFraction, &res)
+	return
+}
+
+// SlashWindow returns # of vote period for oracle slashing
+func (k Keeper) SlashWindow(ctx sdk.Context) (res int64) {
```

Comment:
Does this need to be stored in the keeper? Could this not just be accessed through the paramkeeper?